### ISSUE TITLE
Remove unneeded MechJeb autopilot features from CMA

### ISSUE THREAD

**Issue by sumghai** [sumghai/SDHI_ServiceModuleSystem#127]
As suggested by KSP forums user StevieC The following MechJeb features are not needed from the Crew Module Adapter (CMA), and should be disabled/removed:

- Rover Autopilot
- Spaceplane Guidance
- Landing Guidance

**Comment by sumghai** [comment]
After some experimenting, I have determined that this isn't feasible.

Technically speaking, MechJeb does allow blacklisting of selected features, and in theory I was able to disable all three modules.

However, MechJeb (by default) ships with its own MM patch that adds all MechJeb features to all Command Pods in-game (`GameData\MechJeb2\Parts\MechJebNoCommandPod.cfg`), which always enables Rover, Landing and Spaceplane Autopilot/Guidance.

Since SDHI SMS is designed specifically to be used with the stock Mk1-3 Command Pod, one ends up with a standard SDHI CSM vessel where the Pod has Rover, Landing and Spaceplane Autopilot/Guidance enabled, while the SDHI SMS CMA has the same three features disabled. The net result is that the entire vessel will always have the unwanted MechJeb features no matter what one does.

I could, of course, theoretically write a patch that overrides the MechJeb default and forcibly remove the unwanted autopilots from the Mk1-3 Pod. However, this may cause problems for people who also want to simultaneously use just the Pod in other crafts as rover or lander cabins.

In conclusion, the simplest way to handle all possible use and edge cases is to do nothing.
